### DISCUSSION TITLE
Como conhecer pessoas em eventos / comunidades novas?

### DISCUSSION THREAD

**Discussion by ricdfernandes9** [frontendbr/forum#660] [Discussão]
Olá!

Eu sempre tive essa dúvida mas não consegui ainda encontrar uma boa resposta. Eu acredito que tenha algum problema para interagir com as pessoas de modo geral. Eu já fui para alguns eventos, de nordeste a sul do país, vi palestras de caras que sempre leio seus blogs e códigos no github, mas algumas coisas acontecem:

1. bate a insegurança sobre o que falar com aquele cara... pode parecer besta, mas é algo como se fosse falar com uma mina que vc tá afim. a única coisa que tu n quer é falar besteira.

2. talvez seja impressão minha mas algumas comunidades em alguns eventos são meio fechadas a só quem já é conhecido. tipo, se tu vai pra um estado novo e vai a primeira vez pra um evento, como foi o meu caso, todo mundo lá se conhece, bate papo e tudo mais mas essas mesmas pessoas não fazem com que vc se sinta super bem-vindo logo de cara.... novamente, pode ser impressão minha, mas conversando com alguns colegas de fora do BR eles tbm passam o mesmo lá. Entendem? Como vc vai chegar numa roda de amigos já conhecidos do nada?

3. quando já conheço algumas poucas pessoas em alguns eventos eu me limito a estar com elas e isso não é ruim, só é limitador pq vc acaba não aumentando o networking...

Mas eu percebi algo: uma alternativa é eu meter a cara na doida mesmo e tentar fazer uma talk ou algo do tipo. Até já fiz onde morava mas não adiantava muito ou mesmo, por eu estar nervoso, acabava passando meio despercebido. Mas de qlqr forma, n é todo novato num evento e estado totalmente diferente que vai logo submeter uma palestra... eu acho.

O que eu queria saber é se é assim mesmo, como foi o processo de vocês, o que eu posso fazer pra conseguir mais entrosamento. Perceberam que isso pode ser um problema comigo, com algumas comunidades / eventos ou mesmo os dois? Mas enfim, esse é meu pseudo-desabafo. Na vdd não só meu mas dos meus colegas que percebi que tbm passam por isso em suas diferentes regiões.

Mais alguém já se sentiu ou se sente assim?

**Comment by gabrieldarezzo** [comment]
Comunicação é uma habilidade, assim como tantas outras é necessário exercer a mesma para melhorá-la ( No sentindo da vergonha / iniciar a talk ).

Já parou pra pensar que a comunidade na verdade é você, eu, o palestrante, o cara que está sentado do seu lado, o cara que pegou o ultimo pão de queijo do Meet Up? 

O que é mais hilario é que existe uma possibilidade muito grande da outra pessoa estar encabulada de trocar ideia tmb e acabar perdendo um network bacana por pura vergonha.


> O que eu queria saber é se é assim mesmo, como foi o processo de vocês, o que eu posso fazer pra conseguir mais entrosamento.

Eu comecei pelo digital mesmo, tentando ser mais ativo na comunidade seja respondendo duvidas em fóruns, escrever e compartilhar oq eu estou aprendendo no momento.
Sobre submeter a palestras, acredito que é um segundo passo, eu prefiro ter uma noção legal daquilo ou case interessante antes submeter independente da região, o Zeno fala bastante sobre palestras no DevNaEstrada ( ~ 29m ), recomendo:
https://soundcloud.com/devnaestrada/devnaestrada-63-entrevista-zeno-rocha

**Comment by yanmagale** [comment]
Olá @ricdfernandes9 . Muito legal a sugestão do tema, acredito que seja um tema que várias pessoas tem dúvidas. Vou responder aos tópicos relatando algumas experiências que vivi.

> 1. bate a insegurança sobre o que falar com aquele cara... pode parecer besta, mas é algo como se fosse falar com uma mina que vc tá afim. a única coisa que tu n quer é falar besteira.

Essa sensação é bem comum, eu lembro quando eu conheci o Vedovelli no último BrazilJS e me deu uma sensação de nervosismo sabe. Fiquei pensando: "Nossa, é ele mesmo... o que eu vou falar com ele?". Eu sempre acompanhava(e acompanho) os casts e artigos dele então era uma oportunidade única para dar um __oi__ e poder conversar com ele pessoalmente, saber as opiniões dele sobre algum tema específico em etc. Ele foi super simpático e conversamos bastante sobre diversas coisas, foi muito muito legal. 

Uma das coisas mágicas que acontecem em eventos é justamente essa sinergia de conhecer pessoas e troca de experiências. As pessoas são super solícitas e dispostas a te conhecer e a trocar experiências. Meu conselho seria: __fique tranquilo__ :smile:

<hr>

 > 2. talvez seja impressão minha mas algumas comunidades em alguns eventos são meio fechadas a só quem já é conhecido. tipo, se tu vai pra um estado novo e vai a primeira vez pra um evento, como foi o meu caso, todo mundo lá se conhece, bate papo e tudo mais mas essas mesmas pessoas não fazem com que vc se sinta super bem-vindo logo de cara.... novamente, pode ser impressão minha, mas conversando com alguns colegas de fora do BR eles tbm passam o mesmo lá. Entendem? Como vc vai chegar numa roda de amigos já conhecidos do nada?

Vivi isso na pele e foi uma das melhores experiências que tive ate hoje: participei do Devfest NE em 2015 em Recife e eu seria um dos palestrantes. Eu nunca havia ido em Recife, não conhecia ninguém na cidade, não conhecia absolutamente nada. Quando cheguei ao local do evento, vi as pessoas se reunindo, fazendo networking e trocando experiências, mas ninguém conversava comigo. Ainda mais que, assim que eu abrisse a boca, perceberiam claramente que eu sou de MG _uai_. Pensei comigo: __Olha, se eu não conversar com ninguém, não vou conhecer nada na cidade. Então bora conversar com todo mundo, lets Talk \o/__ . Me aproximei de alguns grupos que estavam próximos a mim e comecei a perguntar se eles eram da cidade, onde e com o quê trabalhavam. A medida que a conversa ia acontecendo, fui ficando mais a vontade e não sentindo esta sensação de "estar longe de casa" ou "de não ter com quem conversar". Resultado: foi um dos melhores eventos que eu fui até hoje, conheci muitas pessoas, fiz novos amigos e conheci vários lugares sobre a cidade, mesmo ficando apenas 3 dias. Meu conselho para você: _Seja educado e apenas converse :+1:_.

<hr>

> 3. quando já conheço algumas poucas pessoas em alguns eventos eu me limito a estar com elas e isso não é ruim, só é limitador pq vc acaba não aumentando o networking...

Sim, eu sempre procuro ir a conferências com os meus amigos ou colegas de trabalho, mas é só você procurar conhecer outras pessoas, aproveitando momentos de almoço, coffe-break e happy hour por exemplo.  



Espero ter ajudado e te vejo nos eventos :smile:

**Comment by kivervinicius** [comment]
falando nisso BRJs ta chegando mais uma oportunidade para a galera se conhecer, como será meu primeiro evento "gigante" estou na mesma que vc

**Comment by ricdfernandes9** [comment]
Obrigado pelas resposta... eu só sinceramente esperava mais participantes na issue rsrs. De qualquer forma obrigado pelo espaço e pelo feedback, @yanmagale e @gabrieldarezzo. Se mais alguém quiser compartilhar algo, será super bem-vindo.

**Comment by alinebastos** [comment]
@ricdfernandes9 calma, a issue só tem dois dias, tem outras que duram dias, semanas.. hehe.
Bom, tu comentou sobre a comunidade ser fechada. Na verdade as pessoas, individualmente, são na sua maioria mais fechadas, tímidas. Da mesma forma que tu tem receio de chegar em alguém, os outros também tem.
Vou dar a mesma dica que já foi dada ali em cima, grupos online!
Entra em grupos de telegram, slack, faz amizades ali, participa dos papos, ajuda pessoas, pede ajuda. Isso já te ajuda a ir conhecendo pessoas da área, aí na hora do evento é mais fácil chegar na pessoa "oi, tu é o fulano lá do grupo tal, né?". Provavelmente o fulano também vai estar louco pra conversar com alguém.
Vou te contar um pouco da minha experiência. Como mulher é mais difícil ainda! Lógico que isso tá melhorando bastante, mas há 5 anos se eu chegasse num cara num evento "oi, tudo bem?", o que o cara ia pensar, né?
Eu comecei a ir em eventos sem conhecer ninguém, entrava muda e saía calada. Era muito ruim, pois os caras ficavam me olhando, eu era quase a única mulher nos eventos, me sentia muito mal. Fazer amizade ali? Quase impossível!
Então tive a sorte de ter colegas de empresa que iriam no BrazilJS de 2012. Já me ofereci pra hospedar eles. Pronto, já consegui fazer amizade com um grupo legal.
O resto fui conhecendo online mesmo!
Entro em todos grupos que posso, adiciono amigos dos meus amigos no face, sigo no twitter. Agora é bem comum alguém chegar e perguntar se eu sou a Aline em algum evento. Pq eu participo de tudo!
Então essa é a minha principal dica, entra em grupos do slack, do telegram, e participa bastante! Vai ser bem mais fácil chegar nas pessoas nos eventos, ou elas mesmas chegarem em ti. Vai ser bem mais fácil falar com aquele palestrante, pois provavelmente ele também vai estar nos grupos que tu participa.
Mas a tua ideia de palestrar é ótima também!! Vai com tudo :)

**Comment by williandutras** [comment]
Opa, beleza @ricdfernandes9 😄

Cara, a pouco tempo eu sofria o mesmo problema que tu, sempre quando eu ia em eventos não conseguia fazer networking e acabava não aproveitando o máximo do evento por causa da timidez. Mas acho que o meu problema era mais ligado a minha autoconfiança, eu morria de medo de não entender o que iria ser falado em grupos, o que responder se caso alguém me perguntasse algo, etc.

O que me fez melhorar muito, foi ficar imerso aos estudos e ficar ativo em grupos de slack, fóruns e até mesmo nas comunidades do facebook. Comecei também a ir em eventos que não eram da área de TI, porque isso me desprendia do medo dos assuntos técnicos, e também fiquei bem mais ativo no meu local de trabalho, ajudando mais aos meus colegas, escrevendo procedimentos da área de front-end pra wiki da empresa. Resumindo, passei a ser mais visto pelas pessoas por justamente está sempre falando e sendo ativo.

Bom, o meu next step agora é seguir a ideia de palestrar. Adorei a ideia, bora!?

Te vejo nos eventos 😄😄

**Comment by gabpaesschulz** [comment]
Poderiam recomendar alguns grupos do facebook/slack? A maioria que vejo é muito pouco movimentado, os posts em sua grande maioria são ou de divulgação de vaga ou de venda de cursos.

**Comment by giioohbernini** [comment]
@gabrielpaesschulz 

Da uma olhada [aqui](http://listatelegram.github.io/)

Todos os grupos que eu conheço/participo e vários outros estão listados ai! É tipo um awesome de grupo no telegram :D

**Comment by alinebastos** [comment]
Olha, tô em vários grupos do Telegram e do Slack, mas o mais movimentado é o grupo do Training Center do slack. É um grupo criado pra pessoas de todas as áreas da TI, tanto pra quem tá iniciando, quanto pra quem já é da área, o pessoal todo se ajuda, tem projetos de mentoria, grupos de estudos, combinações de encontros, muitas ideias de projetos open source, etc.
Entra aqui https://ctgroups.herokuapp.com/ :)

**Comment by gabpaesschulz** [comment]
Boa lista @giioohbernini, já entrei em vários grupos do meu interesse. Agradeço também pela contribuição Aline.

**Comment by leobetosouza** [comment]
Minha dica é sempre: "fala com as pessoas na cara de pau" :)

Eu no início ficava nesse negócio de ficar acanhado, ou com medo de icomodar as pessoas nos eventos. Eu era doido pra falar com o Maujor por exemplo e tinha vergonha, receio, medo... sei lá, é meio estranho de explicar hehe. Foram anos passando pelo Maujor em eventos e não falando com ele. Até que um dia eu tomei coragem, e pra minha surpresa ele já me conhecia, pq eu era bem ativo na comunidade. Foi engraçado.

A dica da @alinebastos de ser ativo online ajuda bastante.

No mais: nunca vi ninguém ficar chateado pq uma outra pessoa num evento chegou e falou "oi, você é o fulano? gosto muito das sus contribuições pra comunidade". Todo mundo sempre fica feliz, por mais experiente e por mais tempo que a pessoa esteja nessa vida, é legal ter alguém que gosta do que você faz. :)

**Comment by synga** [comment]
Olha normalmente o que eu faço é ser bem ativo na comunidade. Ficar no github tentando ajudar nas issues e abrindo novas, no Stackoverflow respondendo a galera, em grupos do Facebook e conversando com a galera no Twitter, comentando posts no LinkedIn.

Mas pra mim a melhor coisa é o Twitter para contatos. Meu primeiro evento foi o JSExperience que teve esses tempos em Sampa, acabei não conhecendo muita gente, mas peguei muitos users de Twitter e por lá interajo com a galera, pergunto, respondo, comento e etc. Fora a galera daqui faço o mesmo com a galera da gringa, por exemplo trampo com Ionic, então sigo os criadores e a galera mais influente na área, sempre conversando, trocando ideias, dando opinião. Então minha primeira aposta é sempre conhecer e conversar no twitter.

O Stackoverflow e grupos no Facebook tem sido minha segunda aposta, mas mais pra ser mais conhecido na área. É bem bacana receber inbox e e-mail da galera com duvidas, sugestões ou elogios, querendo ou não isso cria um network bacana por que alguém sempre vai lembrar o seu nome.

Acho que quanto suas observações, é normal você chegar num local novo e se sentir meio excluído por que realmente a galera desses eventos sempre estão juntas, se vêem em todos os eventos possíveis, mas talvez o seu medo com gente nova seja o mesmo que o deles hahaha.

De qualquer forma aposte em Twitter, alguém sempre vai falar "Caramba, você é o @ tal lá né? Bacana cara" e o papo vai fluir mais de boa.

**Comment by felipefialho** [comment]
@synga Twitter é de longe a melhor rede social
